### ISSUE TITLE
add base/ascii85 unpacker

### ISSUE THREAD

**Issue by jstucke** [fkie-cad/fact_extractor#20]
ascii85 was found in samples (e.g. `cafc32c77a1d7a0fc9ec939219be8d097cf84104f8382fa0425159f46f402192_37834625`) and currently there is no unpacker

**Comment by 0xricksanchez** [comment]
Decoding can be realized via the standard python library. However Ascii85 and Base85 use a different character set so we would need both.

```python
# Default ASCII85 Encoding
In [5]: base64.a85encode(b'Hello World!')
Out[5]: b'87cURD]i,"Ebo80'

# ADOBE ASCII85 Encoding
# Interesting for e.g.: .ps or .eps files especially in printer firmware
In [4]: base64.a85encode(b'Hello World!', adobe=True)
Out[4]: b'<~87cURD]i,"Ebo80~>'

In [6]: base64.b85encode(b'Hello World!')
Out[6]: b'NM&qnZy;B1a%^NF'
```

Additionally there is a `Z85` encoding (see: https://rfc.zeromq.org/spec:32/Z85/), which is a derivative of ASCII85. It is not yet in the `base64` standard python library (see: https://bugs.python.org/issue31116).

**Comment by jstucke** [comment]
resolved in #28
